src/components/Chat/Chat.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import './Chat.styl';
+import Message from './Message/Message';
+import Textarea from 'react-textarea-autosize';
+
+class Chat extends Component {
+    constructor(props) {
+        super(props);
+        this.sendMessage = this.sendMessage.bind(this);
+        this.onKeyDown = this.onKeyDown.bind(this);
+        this.state = {
+            value: '',
+        };
+    }
+    static name() {
+        return 'Chat';
+    }
+    sendMessage() {
+        let elem = {
+            author: 'test',
+            content: this.state.value,
+            date: (new Date()).toLocaleTimeString(),
+        };
+        this.setState({
+            value : '',
+        });
+        this.props.onMessage(elem);
+    }
+    onKeyDown(event) {
+        if (event.keyCode == 13 && event.ctrlKey) {
```

Comment:
`if (event.keyCode === 13 && event.ctrlKey && this.state.value !== '') this.sendMessage()`